pkg/action/install.go

```diff
@@ -174,7 +173,7 @@ func (i *Install) installCRDs(crds []chart.CRD) error {
 	}
 	if len(totalItems) > 0 {
 		// Give time for the CRD to be recognized.
-		if err := i.cfg.KubeClient.Wait(totalItems, 60*time.Second); err != nil {
+		if err := i.cfg.KubeClient.Wait(totalItems, i.Timeout); err != nil {
```

Comment:
Strictly, this is a behavior change. A user with a very fast to deploy change, might have e.g. `--timeout=5s`. With the understanding that CRDs would wait for a minute.

I think we should continue the existing 60s (as minimum), and only allow the timeout to be extended: `max(60*time.Second, i.Timeout)`

Reply:
The alternative, would be to introduce a new command line flag / API option. Personally, I feel like this is unnecessary here (but that is up for discussion)

Reply:
Hi @gjenkins8  
I feel that what you said about a minimum of 60 seconds is indeed correct. It does make more sense logically.

Reply:
This is indeed logically reasonable, but I found that many reused timeouts do not have a fixed minimum value.
![image](https://github.com/helm/helm/assets/112474703/addfe04b-d73c-48d3-9ae9-cccd3b6289b4)

![image](https://github.com/helm/helm/assets/112474703/bbacc397-10d3-4e24-9efe-da3143e37c24)